doc/processing.md

```diff
@@ -0,0 +1,92 @@
+## How OAR Facilities Are Created And Matched
+
+The list of searchable facilities in the Open Apparel Registry is compiled from
+contributor submitted lists. The application uses statistical text analysis to
+compare newly submitted items to the existing list of facilities in an attempt
+to automatically match highly similar items and to suggest potential matches
+that require confirmation when there weaker similarities between items.
+
+Contributors submit their facility lists to the Open Apparel registry for
+processing by uploading lists in comma-separated value (CSV) format. All popular
+spreadsheet programs, including Microsoft Excel, can save a sheet in CSV format.
+The Open Apparel Registry provides a downloadable template, but any CSV can be
+read and processed if it meets the following criteria:
+
+  - Includes a header row with the following field names (in lower case)
+    - country
+    - name
+    - address
+  - Less than 5MB in size.
```

Comment:
May be worth enumerating another criterion here which is something like

```
- Has facility rows made up of comma separated country, name, and address fields
```

Reply:
Fixed in 4d2c0ab

Reply:
We may have decided to drop this change, but I don't see it in the updated version either.